src/test/java/org/unicode/cldr/surveydriver/SurveyDriverData.java

```diff
@@ -13,6 +13,8 @@ public class SurveyDriverData {
      * in PathHeader.java, and inserting this line into the PageId initialization function:
      * System.out.println("PageId raw name: " + this.toString());
      *
+     * It would be better to have an API for requesting this list from the ST back end at runtime!
```

Comment:
that's the best idea.  I think there should be an API for serializing the entirety of PathHeader, so that it can be calculated on the client side.